indico/modules/categories/controllers/management.py

```diff
@@ -34,3 +35,15 @@ def _category_query_options(self):
     def _process(self):
         return WPCategoryManagement.render_template('management/content.html', self.category, 'content',
                                                     categories=self.category.children)
+
+
+class RHManageSubcategories(RHManageCategoryBase):
+    def _checkParams(self):
+        RHManageCategoryBase._checkParams(self)
+        self.subcategories = Category.find().filter(Category.id.in_(request.json['categories'])).all()
```

Comment:
btw, `find()` without arguments doesn't make much sense. you'd use the `query` property in that case. the only purpose of `find()` is to provide some sugar (it accepts *args and **kwargs like filter/filter_by and some special ones). but even though i added it initially i'm more and more tempted towards using `.query.filter(..)` and `.query.filter_by()` in favor of `.find()` simply because it's clearer and more like standard sqlalchemy